articles/fleetctl.md

```diff
@@ -20,6 +20,16 @@ sudo npm install -g fleetctl
 
 To install fleetctl on Windows or Linux, download the fleectl binary here on [GitHub](https://github.com/fleetdm/fleet/releases).
 
+### Additional dependencies
+
+To use `fleetctl package` for platforms other than Windows, you'll need to run `fleetctl` from macOS or Linux.
```

Comment:
What does this mean as a new Fleet user? Fleet's agent for macOS and Linux can only be generated on macOS or Linux?